Header header
velodyne/DataChunkMsg[12] dataChunks
uint16 spinCount
uint32 reserved
